### PR TITLE
change default to false

### PR DESCRIPTION
**Pull request by yunfeiguo**
@bayolau

### CODE REVIEW

File: simulate.py [hunk 1]
```diff
@@ -59,7 +59,8 @@ def get_contigs(self):
     parser.add_argument("--max_frag", help="Maximum length of fragment", type=int, default=1000000000)
     parser.add_argument("--min_pass", help="Minimum passes", type=int, default=1)
     parser.add_argument("--max_pass", help="Maximum passes", type=int, default=1000000000)
-    parser.add_argument("--output_polymerase_read", help="shall we simulate polymerase read (including adapter)?", action='store_false')
+    parser.add_argument("--output_polymerase_read", help="shall we simulate polymerase read (including adapter)?", action='store_true')
+    parser.set_defaults(output_polymerase_read=False)
     parser.add_argument("--adapter_sequence", type=str, help="specify adapater sequence for polymerase reads.", default="")
     parser.add_argument("--scaled_median_frag", help="scale fragment length to this value", type=int, default=-1)
     parser.add_argument("--custom_rate", help="i:d:s:m, where i/d/s/m are integer-frequency of insertion/deletion/substitution/match. For example, 0:0:0:1 means perfect sequencing.", type=str, default=None)
```
Comment (marghoob) [inline simulate.py]: type is str by default, anyway.

File: simulate.py [hunk 2]
```diff
@@ -59,7 +59,8 @@ def get_contigs(self):
     parser.add_argument("--max_frag", help="Maximum length of fragment", type=int, default=1000000000)
     parser.add_argument("--min_pass", help="Minimum passes", type=int, default=1)
     parser.add_argument("--max_pass", help="Maximum passes", type=int, default=1000000000)
-    parser.add_argument("--output_polymerase_read", help="shall we simulate polymerase read (including adapter)?", action='store_false')
+    parser.add_argument("--output_polymerase_read", help="shall we simulate polymerase read (including adapter)?", action='store_true')
+    parser.set_defaults(output_polymerase_read=False)
```
Comment (yunfeiguo) [inline simulate.py]: Thanks! I didn't realize that until I pushed the commit. Shall I PR again? @marghoob

Comment (marghoob) [inline simulate.py]: It's fine for now. Just keep this in mind for the future.

**Comment by yunfeiguo** [comment]
Sure. Thank you!

**Pull request merged**